spark/spark-base/src/main/java/com/thoughtworks/go/spark/Routes.java

```diff
@@ -362,6 +362,20 @@ public static String id(String id) {
         }
     }
 
+    public static class PluginInfoAPI {
+        public static final String BASE = "/api/admin/plugin_info";
```

Comment:
Why is this under `admin` namespace?
the plugin info API is accessible by all users.

\cc: @maheshp @ketan